bindings/node.js/kzg.ts

```diff
@@ -13,6 +13,7 @@ export type Blob = Uint8Array; // 4096 * 32 bytes
 
 type SetupHandle = Object;
 
+export type ComputationProof = [KZGProof, Bytes32];
```

Comment:
I wasn't sure what to name this and am curious if it's ok.

Reply:
Perhaps ProofResult?